From 7357320b006040f039f6c73da96fa7e5bba57a07 Mon Sep 17 00:00:00 2001
From: caoyi1 <caoyi1@lenovo.com>
Date: Thu, 31 Dec 2015 15:27:10 +0800
Subject: [PATCH 591/959] IKQCOMSD1-3111:input misc epl8802,update calibration
 parameter

Change-Id: I3c288c63b8f5c5850d3299b89ba074f968740a62
Signed-off-by: caoyi1 <caoyi1@motorola.com>
Reviewed-on: http://gerrit.mot.com/810268
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Robert Wiegand <Robert.P.Wiegand@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 drivers/input/misc/epl8802.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/input/misc/epl8802.c b/drivers/input/misc/epl8802.c
index 81277ac8f99..9aebda0fc2c 100644
--- a/drivers/input/misc/epl8802.c
+++ b/drivers/input/misc/epl8802.c
@@ -85,8 +85,8 @@
 #define ALS_LOW_THRESHOLD		1000
 #define ALS_HIGH_THRESHOLD		3000
 
-#define PS_LOW_THRESHOLD		500 // PS low threshold
-#define PS_HIGH_THRESHOLD		800 // PS high threshold
+#define PS_LOW_THRESHOLD		2073 /*PS low threshold*/
+#define PS_HIGH_THRESHOLD		7525 /*PS high threshold*/
 
 #define LUX_PER_COUNT			400 //ALS lux per count, 400/1000=0.4
 #define ALS_LEVEL    16
@@ -233,8 +233,8 @@ int epl_sensor_als_dyn_report(bool report_flag);
 #if PS_DYN_K_ONE
 static bool ps_dyn_flag = false;
 #define PS_MAX_CT	10000
-#define PS_DYN_H_OFFSET 800
-#define PS_DYN_L_OFFSET 500
+#define PS_DYN_H_OFFSET 6943
+#define PS_DYN_L_OFFSET 2024
 u32 dynk_thd_low = 0;
 u32 dynk_thd_high = 0;
 #endif
-- 
2.11.0

